packages/nextjs/app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+
+const umutArray:NextPage = () => {
```

Comment:
There's still one pesky Lint error related to a missing symbol on this line (careful, that's not a period it's talking about)
```
./app/builders/0x3566ce434c0d8B80934354c806f8651e1cC91EDD/page.tsx
5:17  Warning: Insert `·`  prettier/prettier
```

Make sure you run `yarn build` (while your terminal is in the `nextjs` folder) so you can catch these little errors _before_ you push commits.